src/tools/quemap/faces.c

```diff
@@ -441,10 +442,15 @@ static void FixEdges_r(node_t *node) {
 	}
 }
 
+void FixTJunctionsQ3( node_t *node );
+
 /**
  * @brief
  */
 void FixTjuncs(node_t *head_node) {
+	FixTJunctionsQ3(head_node);
+	FixTJunctionsQ3(head_node);
+
```

Comment:
It looks like we're always calling into the Q3 code, regardless of the `notjunc` parameter?

Reply:
Er, and calling this twice?